cuda/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -325,6 +325,34 @@ TEST_F(Jacobi, CudaPreconditionerEquivalentToRefWithMPW)
 }
 
 
+TEST_F(Jacobi, CudaTransposedPreconditionerEquivalentToRefWithMPW)
+{
+    initialize_data({0, 11, 24, 33, 45, 55, 67, 70, 80, 92, 100}, {}, {}, 13,
+                    97, 99);
+
+    auto bj = bj_factory->generate(mtx);
+    auto d_bj = d_bj_factory->generate(mtx);
+    d_bj->copy_from(bj.get());
+
+    GKO_ASSERT_MTX_NEAR(gko::as<Bj>(d_bj->transpose()),
+                        gko::as<Bj>(bj->transpose()), 1e-13);
```

Comment:
use 1e-14 in omp but 1e-13 in cuda/hip. Does it fail when using 1e-14?

Reply:
no, 1e-14 still works :)